tsconfig.json

```diff
@@ -5,7 +5,9 @@
         "noImplicitAny": false,
         "sourceMap": true,
         "alwaysStrict": true,
-        "outDir": "./"
+        "outDir": "./",
```

Comment:
`types/` is already in `.gitignore` and I would prefer to use paths rather than specific file extensions.